### ISSUE TITLE
Could you please provide bom

### ISSUE THREAD

**Issue by cxp297** [makerbase-mks/MKS-TinyBee#23]
Could you please provide bom

**Comment by rsuzano** [comment]
Please, I need to... My thermistor short circuited at heatblock and fried D7 diode, it's part of power circuit, everyting works, but the D7 diode have a hole on it ...and I think this diode it's some kind of protection.
I only find this picture with a baddly D7 label reading... but I don't find any equivalent on google

![image](https://user-images.githubusercontent.com/10420338/167165040-ccddfd28-effc-44de-89fa-11ae1172b030.png)

**Comment by jirka-malvin** [comment]
all BOM are available at github. Just look up.

**Comment by irazaq** [comment]
@jirka-malvin that is not helpful, you could at least give a link to the BOM for tinybee. I am also looking for it too,  there is no BOM file in the hardware directory. 
![makerbase-mks_MKS-TinyBee](https://user-images.githubusercontent.com/431541/190924329-8c1a0df9-7608-40b5-9fab-3822de86e441.png)

**Comment by jirka-malvin** [comment]
@irazaq , You are right, I was wrong - BOM is only at first sheet and only partly (capacitors). BOM is not complete at all.

**Comment by SubJORJ** [comment]
This is a Zener diode of approximately 2.5~2.7V. The accuracy of readings of temperature sensors TH1, TH2, TB depends on it.
When it failed for me, the temperature sensors began to lie by about -40 degrees.
